### PR TITLE
Simplified Grids

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
1. Two items in a grid.
2. Three items in a gri
Screenshot (346)
Screenshot (347)

### CODE REVIEW

File: components/Grids/Grids.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.two-box-grid{
+    grid-template-columns: 1fr 1fr;
+}
+.three-box-grid{
+    grid-template-columns: 1fr 1fr 1fr;
+}
+.grid{
+    gap: 5%;
+    margin-top: 0.5rem;
```
Comment (saadahmadhere) [inline components/Grids/Grids.css:9]: 0.5rem can be written as 8px

**Review by saadahmadhere — COMMENTED** [review]
Code looks good.

File: components/Grids/Grids.css (hunk 1 above)
Comment (shuklahimanshu81) [reply components/Grids/Grids.css:9]: Isn't using rem is better than px for responsiveness.

File: components/Grids/Grids.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+.two-box-grid{
+    grid-template-columns: 1fr 1fr;
+}
+.three-box-grid{
+    grid-template-columns: 1fr 1fr 1fr;
+}
```
Comment (Tanay-J) [inline components/Grids/Grids.css:6]: Approving the changes.
Just a suggestion: class names can be improved, you may have a look at tailwind grid classnames.

**Review by Tanay-J — APPROVED** [review]

File: components/Grids/Grids.css (hunk 2 above)
Comment (shuklahimanshu81) [reply components/Grids/Grids.css:6]: will look into them for sure. Thanks for the suggestion Tanay.

**Pull request merged**